cmd/tether/attach.go

```diff
@@ -437,8 +446,15 @@ func (t *attachServerSSH) run() error {
 			log.Debugf("reader/writers bound for channel for %s", sessionid)
 
 			go t.channelMux(requests, session, detach)
-		}
 
+			// FIXME: This needs to be runblock instead of Attach
+			if session.Attach && session.Started != "true" {
+				// make sure that portlayer received the container id back
+				log.Debugf("Unblocking the launch of %s", sessionid)
+				session.ClearToLaunch <- <-t.askedAndAnswered
```

Comment:
Does this work? Would be able to just do this with one channel that you close below and other places we just wait for that?